docker-config/os-packager/Dockerfile

```diff
@@ -1,12 +1,17 @@
-FROM gliderlabs/alpine:latest
+FROM gliderlabs/alpine:3.3
+
 
 RUN apk add --update python
+RUN apk add --update --virtual=build-dependencies ca-certificates build-base
 RUN apk add git
 RUN apk add nodejs
 RUN git clone http://github.com/openspending/os-packager.git app
+RUN npm install npm -g
+RUN npm --version
 RUN cd app && npm install --only=prod
```

Comment:
TBH I never gave it too much thought... 
but now that I look at the package.json, I don't really follow the logic of the separation between dev and prod dependencies here: https://github.com/openspending/os-packager/blob/master/package.json

e.g. why is jquery & bootstrap a dev dep., but underscore and d3 not? or angular a dev dep., but angular-route not?

Perhaps @kravets-levko can shed some light here?